service/src/test/java/io/camunda/service/AuthorizationServiceTest.java

```diff
@@ -16,23 +16,31 @@
 import io.camunda.search.filter.AuthorizationFilter;
 import io.camunda.search.query.SearchQueryBuilders;
 import io.camunda.search.query.SearchQueryResult;
+import io.camunda.security.configuration.SecurityConfiguration;
 import io.camunda.service.security.SecurityContextProvider;
 import io.camunda.zeebe.broker.client.api.BrokerClient;
+import java.util.Set;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class AuthorizationServiceTest {
 
   private AuthorizationServices services;
   private AuthorizationSearchClient client;
+  private SecurityConfiguration securityConfiguration;
 
   @BeforeEach
   public void before() {
+    securityConfiguration = new SecurityConfiguration();
```

Comment:
❓ Do we need to create a new object here or can we use a mock and then use a when in the tests to return the test value vs setting the field

Reply:
What would be the advantage? I didn't see a need to mock as the config POJO is simple enough and doesn't have any dependencies itself, kind of following a mock only when it makes it simpler approach.